src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/PublicationServiceTests.cs

```diff
@@ -2027,6 +2027,97 @@ await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId)
             }
         }
 
+        public class ListSitemapItemsTests : PublicationServiceTests
+        {
+            private readonly string publicationUpdated = "2018-04-06T13:46:11";
+            private readonly Guid publicationId = Guid.NewGuid();
+
+            private readonly Guid firstReleaseId = Guid.NewGuid();
+            private readonly string firstReleaseSlug = "first-release-slug";
+
+            private readonly Guid secondReleaseId = Guid.NewGuid();
+            private readonly string secondReleaseSlug = "second-release-slug";
+
+            private readonly string firstReleaseVersionPublishedDate = "2019-02-03T07:34:12";
+            private readonly string firstReleaseVersionUpdateDate = "2019-02-04T08:29:54";
```

Comment:
If these fields are only used by a single test, then I'd suggest they be declared as part of the test setup, rather than at class-level.

Reply:
Yeah I had the same thought myself, but was trying to be consistent with how we do this elsewhere. I'll move them in